resources/views/index.blade.php

```diff
@@ -14,6 +14,7 @@
     <link rel="stylesheet" type="text/css" href="angular/account/register.css">
     <link rel="stylesheet" type="text/css" href="angular/about/about.css">
     <link rel="stylesheet" type="text/css" href="angular/admin/privacy.css">
+    <link rel="stylesheet" type="text/css" href="angular/home/welcome.css">
```

Comment:
Yikes, we'll need some Gulp action to combine these or a reason they are all loaded but in separate files?